doc/requirements/cleaning

```diff
@@ -0,0 +1,224 @@
+
+1. NOP
+2. ALRU
+3. ACP
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: NOP cleaning policy
+id: nop
+---
+
+When NOP is set as the active cleaning policy the product shall not perform
+background cleaning.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: ALRU wake up interval when IO requests are detected
+id: alru_wake_up_on_io
+---
+
+When ALRU is set as the active cleaning policy and the time passed since the last
+IO request to the cache instance is lower than ACTIVITY_THRESHOLD the product
+shall put the background worker to sleep for WAKE_UP.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: ALRU flush conditions
+id: alru_flush_conditions
+---
+
+When ALRU is set as the active cleaning policy the product shall flush dirty cache
+lines which had not been written for at least STALENESS_TIME.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: ALRU wake up interval when nothing to flush
+id: alru_wake_up_on_no_flush
+---
+
+When ALRU is set as the active cleaning policy and eligible cache lines to flush
+had not been found the product shall put the background worker to sleep for WAKE_UP.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: ALRU max flushed cache lines
+id: alru_max_buffers
+---
+
+When ALRU is set as the active cleaning policy the product shall flush at most
+FLUSH_MAX_BUFFERS dirty cache lines in a single iteration.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: ALRU valid wake up values
+id: alru_valid_wake_up
+---
+
+The product shall accept WAKE_UP values in range from 1 to 3600 inclusive.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: ALRU valid staleness time values
+id: alru_valid_staleness_time
+---
+
+The product shall accept STALENESS_TIME values in range from 1 to 3600 inclusive.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: ALRU valid activity threshold values
+id: alru_valid_activity_threshold
+---
+
+The product shall accept ACTIVITY_THRESHOLD values in range from 0 to 1000000 inclusive.
```

Comment:
I meant not to specify units in a separate requirement for the product. However for this requirement (and any other numeric values in this file) we have to state unit explicitly for it to be verifiable.